README.md

```diff
@@ -1,141 +1,146 @@
+## Governance
 
- - **Upgradability**: The ability to upgrade contracts that constitute the Network. Granting authority to upgrade contracts is a superset of all other privileges and therefore should require the highest possible authority.  
- - **Courts**: The ability to arrange the court hierarchy, and set arbitration fees at the court and network level. Courts are used to enforce agreements within the network, significant changes to the structure and fees of courts should require significant authority.
- - **Monetary Policy**: The ability to manage price feeds, stability targets, and participation targets that govern the supply of ANT and the stability reserve. These parameters are critical to the stable growth of the network, but changes are constrained such that the impact of any single action is minimized.  
- - **Fiscal Policy**: The ability to direct discretionary funds held in the Network's treasury. Since the treasury is isolated from the Network's Stability Reserve, governance of discretionary funds will have limited short-term impact on the network.  
-## 5.2 Stakeholder Representation
-Actions that require high authority, such as upgradeability, should ensure that all stakeholders have some way to check the power of other stakeholders. In the case of an upgrade of the Aragon Network we should consider: users of agreements, ANT holders, and Jurors. While the exact processes for governance of network upgrades has not been defined and will ultimately be decided through a series of experimental deployments, a reasonable process that incorporates checks and balances would look something like this:
+Governance authority over the court is granted to ANT holders by way of an Aragon organization.
 
- 1. ANT holders vote to propose an upgrade
- 2. Reputation holders in the supreme court vote to approve the proposal
- 3. A wait period is enforced before the upgrade occurs that allows parties of agreements to mutually decide to use a different arbitration mechanism, or simply settle their agreement.
-## 5.3 Incentivization
-Every participant benefits from good governance but the effort and opportunity cost of actively contributing leads to minimal participation and poor social outcomes. This is a classic free-rider situation, where the benefits are shared among everyone in the network, but the burden is not widely distributed.
+Initially, all votes will last 1 Month, require 50% approval and 1% approval quorum. Creating votes requires depositing 1000 ANT into a *proposal agreement* with the following terms:
```

Comment:
> and 1% approval quorum

Is "approval quorum" being used as shorthand for "Minimum Acceptance Quorum"?

Reply:
Yes updated to reflect formal language for the voting parameters.